specification/devcenter/data-plane/Microsoft.DevCenter/stable/2023-04-01/devcenter.json

```diff
@@ -2623,6 +2632,10 @@
         "required"
       ]
     },
+    "EnvironmentPatchProperties": {
```

Comment:
In 2024-02-01:
`model EnvironmentPatchProperties` is new, and the existing `model EnvironmentUpdateProperties` is made to extend it as such: `model EnvironmentUpdateProperties is EnvironmentPatchProperties`

The problem with this is that it adds EnvironmentPatchProperties to 2023-04-01

If I add the `@added(APIVersions.v2024_02_01)` versioning decorator to `EnvironmentPatchProperties`, I then get an error saying I need to add it to `EnvironmentUpdateProperties` as well, and then doing that means it will delete it from this file. 

Don't have a solution to this at the moment.

Reply:
oh i see. can you add the api version to patch if `EnvironmentUpdateProperties extends EnvironmentPatchProperties`?  If not we need to open an issue so they can fix that

Reply:
Closing this comment out since 
https://github.com/Azure/azure-rest-api-specs/pull/28531#discussion_r1544941816
tracks the same issue.